src/services/motionModel.ts

```diff
@@ -7,21 +7,25 @@ import {
   packMetadata,
   selectImages,
   syncCursors,
-  MAX_FAILED_ITERATIONS,
+  MAX_FAILED_ITERATIONS, getConfig,
 } from 'util/motionModel';
 import { FramesMetadata, GnssMetadata } from 'types/motionModel';
 import { promiseWithTimeout, sleep } from 'util/index';
 import { concatFrames } from 'util/framekm';
-import { rmSync } from 'fs';
-import { MOTION_MODEL_CURSOR } from 'config';
+import { existsSync, mkdir, rmSync } from 'fs';
+import { MOTION_MODEL_CURSOR, RAW_DATA_ROOT_FOLDER } from 'config';
 import { ifTimeSet } from 'util/lock';
 import { isIntegrityCheckDone } from './integrityCheck';
 import { isCarParkedBasedOnImu } from 'util/imu';
 import { Instrumentation } from 'util/instrumentation';
+import { getRawImuData, writeRawData } from 'util/datalogger';
+import console from 'console';
 const ITERATION_DELAY = 5400;
 
 export const lastProcessed = null;
 let failedIterations = 0;
+let lastTimeRawSnippetCreated = Date.now();
+const config = getConfig();
```

Comment:
@punov One thing that I am not sure, is this line here.
I added the `getConfig()` to fetch it, but I not sure if this config will get _overridden_ and not require a `systemctl stop/start camera-node`